editor/test/integration/test_util.clj

```diff
@@ -110,9 +113,11 @@
   ([graph]
    (setup-workspace! graph project-path))
   ([graph project-path]
-   (let [workspace (workspace/make-workspace graph
+   (let [workspace-config (shared-editor-settings/load-project-workspace-config project-path)
+         workspace (workspace/make-workspace graph
                                              (.getAbsolutePath (io/file project-path))
-                                             {})]
+                                             {}
+                                             workspace-config)]
```

Comment:
The `workspace/make-workspace` function now takes a `workspace-config` map where you can customize the resulting workspace. Supplying a vector of proj-paths as `:non-editable-directories` will affect how resources below those directories are loaded and connected into the graph.

This change to `test-util/setup-workspace!` ensures the tests respect the `workspace-config` in `proejct.shared_editor_settings` if present.